### PR TITLE
created blog post form

### PR DESCRIPTION
**Pull request by sanixstudio**

### CODE REVIEW

File: client/src/containers/PostForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		<NewPost>
+    	<NewPost.FormHeading>Create New Post</NewPost.FormHeading>
+    	<NewPost.FormControler isRequired>
+      	<NewPost.FormLabeling>Topic</NewPost.FormLabeling>
+      	<NewPost.FormSelect placeholder="Select Topic">
+      	<option value="option1">Front-end</option>
+      	<option value="option2">Back-end</option>
+      	<option value="option3">UX/UI</option>
+      	<option value="option3">Database</option>
+      	<option value="option3">Design</option>
```
Comment (awwmicky) [inline client/src/containers/PostForm/index.jsx:14]: convert these components into compound

**Review by awwmicky — CHANGES_REQUESTED** [review]
all the work is looking good! just update the `<option>` tag

File: client/src/containers/PostForm/index.jsx (hunk 1 above)
Comment (sanixstudio) [reply client/src/containers/PostForm/index.jsx:14]: It will be done

**Review by awwmicky — APPROVED** [review]
looking sharp & clean! thanks for the update

**Pull request merged**